### ISSUE TITLE
Rust Tooling Missing ERR Msg - using VS-Code

### ISSUE THREAD

**Issue by RobFosterNYC** [rust-lang/rust#101117]
Rust Tooling Missing ERR: message - It constantly nags me to install **racer**,
But I was told not to install racer by friends who are trying to teach me Rust.
I constantly get windows popup reminding me,  every time I manipulate the code or save it or start VS-Code.

The ENV Path is set properly, and I have all the proper files installed in the Cargo folder in Cargo/bin
I don't know why this keeps nagging me, but its very discouraging when trying to learn Rust.  
thank you for your time. :)

![Screenshot_2](https://user-images.githubusercontent.com/18384135/187065662-0696e076-a1d1-4080-ac57-10b29640fd75.jpg)

**Comment by ChayimFriedman2** [comment]
You want the rust-analyzer extension, not the rust extension.

**Comment by ChayimFriedman2** [comment]
rust-analyzer and the Rust extension don't work well together. Remove/disable the second. I said that because the popup says it is from the Rust extension.

**Comment by RobFosterNYC** [comment]
My friend fixed it.  
It seems a long long time ago I installed the Rust extension and when I uninstalled it because it got deprecated,
some files were still left behind for some weird and odd reason.

I left the Rust Analyzer installed as Rust website says to have it installed as well as many videos I have come across.
It works now because a friend of mine came over and fixed it. 
He has way more experience than me, around 20 years as a Dev & found out what was wrong.
